### ISSUE TITLE
Migration doesn't works

### ISSUE THREAD

**Issue by aluco100** [typeorm/typeorm#1438]
I run the sample above:

https://github.com/typeorm/typeorm/tree/master/sample/sample32-migrations

But I got the following error when I'm rename the property `name` to `title` from `Post` entity:

```
executing query:  BEGIN TRANSACTION
executing query:  SELECT * FROM sqlite_master WHERE type = 'table' AND name IN ('author', 'post')
executing query:  PRAGMA table_info("author")
executing query:  PRAGMA index_list("author")
executing query:  PRAGMA foreign_key_list("author")
executing query:  PRAGMA table_info("post")
executing query:  PRAGMA index_list("post")
executing query:  PRAGMA foreign_key_list("post")
executing query:  CREATE TABLE "temporary_author" ("id" integer NOT NULL PRIMARY KEY AUTOINCREMENT, "firstName" varchar NOT NULL,
"lastName" varchar NOT NULL)
executing query:  INSERT INTO "temporary_author"("id", "firstName", "lastName") SELECT "id", "firstName", "lastName" FROM "author"
executing query:  DROP TABLE "author"
query failed:  DROP TABLE "author"
error:  { Error: SQLITE_CONSTRAINT: FOREIGN KEY constraint failed errno: 19, code: 'SQLITE_CONSTRAINT' }

executing query:  ROLLBACK
Error:  { QueryFailedError: SQLITE_CONSTRAINT: FOREIGN KEY constraint failed
    at new QueryFailedError (/Users/alfredoluco/Documents/MyProject2/src/error/QueryFailedError.ts:7:9)
    at Statement.<anonymous> (/Users/alfredoluco/Documents/MyProject2/src/driver/sqlite/SqliteQueryRunner.ts:54:26)
    at Statement.replacement (/Users/alfredoluco/Documents/MyProject2/node_modules/sqlite3/lib/trace.js:19:31)
    at Statement.replacement (/Users/alfredoluco/Documents/MyProject2/node_modules/sqlite3/lib/trace.js:19:31)
  message: 'SQLITE_CONSTRAINT: FOREIGN KEY constraint failed',
  errno: 19,
  code: 'SQLITE_CONSTRAINT',
  name: 'QueryFailedError',
  query: 'DROP TABLE "author"',
  parameters: [] }


```

What could be happen?

Best regards

**Comment by pleerock** [comment]
its hard to debug things this way, I don't have error you are talking about with this sample. Can you create a reproduction git repo where I can check this code and error?

**Comment by aluco100** [comment]
This is my repo:

https://github.com/aluco100/TypeORMMigrationSample

Thanks very much

**Comment by AlexMesser** [comment]
fixed and released in `typeorm@0.2.0-alpha.27`. You can try it via `npm i typeorm@next`.
